<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="load_0">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>460</y>
      <width>54</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="load_1">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>435</y>
      <width>54</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="load_2">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>410</y>
      <width>54</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="load_3">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>360</y>
      <width>54</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="load_4">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>385</y>
      <width>54</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="picture_label">
    <property name="geometry">
     <rect>
      <x>329</x>
      <y>19</y>
      <width>871</width>
      <height>611</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/img/img/tomasulo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="queue_5">
    <property name="geometry">
     <rect>
      <x>618</x>
      <y>90</y>
      <width>99</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="queue_4">
    <property name="geometry">
     <rect>
      <x>618</x>
      <y>115</y>
      <width>99</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="queue_3">
    <property name="geometry">
     <rect>
      <x>618</x>
      <y>139</y>
      <width>99</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="queue_2">
    <property name="geometry">
     <rect>
      <x>618</x>
      <y>163</y>
      <width>99</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="queue_1">
    <property name="geometry">
     <rect>
      <x>618</x>
      <y>188</y>
      <width>99</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="queue_0">
    <property name="geometry">
     <rect>
      <x>618</x>
      <y>212</y>
      <width>99</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="storeAddr_0">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>461</y>
      <width>59</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="storeAddr_1">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>436</y>
      <width>59</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="x_1">
    <property name="geometry">
     <rect>
      <x>434</x>
      <y>188</y>
      <width>69</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="x_2">
    <property name="geometry">
     <rect>
      <x>434</x>
      <y>163</y>
      <width>69</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="x_3">
    <property name="geometry">
     <rect>
      <x>434</x>
      <y>139</y>
      <width>69</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="x_0">
    <property name="geometry">
     <rect>
      <x>434</x>
      <y>212</y>
      <width>69</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="addsOp_0">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>484</y>
      <width>29</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="addsOp_1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>457</y>
      <width>29</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="addsOp_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>430</y>
      <width>29</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="addsVj_0">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>484</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="addsVj_1">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>457</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="addsVj_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>430</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="addsVk_0">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>484</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="addsVk_1">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>457</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="addsVk_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>430</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mulsOp_1">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>450</y>
      <width>29</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mulsVj_0">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>473</y>
      <width>69</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mulsVj_1">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>450</y>
      <width>69</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mulsVk_0">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>473</y>
      <width>63</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mulsVk_1">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>450</y>
      <width>63</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="storeAddr_2">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>411</y>
      <width>59</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="storeAddr_3">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>386</y>
      <width>59</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="storeAddr_4">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>360</y>
      <width>59</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="storeReg_0">
    <property name="geometry">
     <rect>
      <x>828</x>
      <y>461</y>
      <width>69</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="storeReg_1">
    <property name="geometry">
     <rect>
      <x>828</x>
      <y>436</y>
      <width>69</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="storeReg_2">
    <property name="geometry">
     <rect>
      <x>828</x>
      <y>411</y>
      <width>69</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="storeReg_3">
    <property name="geometry">
     <rect>
      <x>828</x>
      <y>386</y>
      <width>69</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="storeReg_4">
    <property name="geometry">
     <rect>
      <x>828</x>
      <y>360</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="memoryTable">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>50</y>
      <width>131</width>
      <height>561</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="tabKeyNavigation">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="portTable">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>620</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
   </widget>
   <widget class="QLabel" name="x_4">
    <property name="geometry">
     <rect>
      <x>434</x>
      <y>115</y>
      <width>69</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="x_5">
    <property name="geometry">
     <rect>
      <x>434</x>
      <y>90</y>
      <width>69</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="xq_0">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>212</y>
      <width>63</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="xq_1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>188</y>
      <width>63</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="xq_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>163</y>
      <width>63</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="xq_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>139</y>
      <width>63</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="xq_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>115</y>
      <width>63</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>660</y>
      <width>586</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="resetButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Reset simulation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Start Simulation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clkButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Clk</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="skipButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Skip 10 clocks</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="clkLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="xq_5">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>63</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mulsOp_0">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>470</y>
      <width>29</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="add_0">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>351</x>
      <y>530</y>
      <width>69</width>
      <height>29</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="add_1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>427</x>
      <y>530</y>
      <width>68</width>
      <height>29</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="add_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>502</x>
      <y>530</y>
      <width>68</width>
      <height>29</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="mul_0">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>624</x>
      <y>520</y>
      <width>69</width>
      <height>29</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="mul_1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>520</y>
      <width>68</width>
      <height>29</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="compileButton">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>510</y>
      <width>291</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Compile program</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="codeTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>22</y>
      <width>291</width>
      <height>481</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>299</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="compileOutput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>291</width>
      <height>101</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>299</width>
      <height>192</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>picture_label</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>load_0</zorder>
   <zorder>load_1</zorder>
   <zorder>load_2</zorder>
   <zorder>load_3</zorder>
   <zorder>load_4</zorder>
   <zorder>queue_5</zorder>
   <zorder>queue_4</zorder>
   <zorder>queue_3</zorder>
   <zorder>queue_2</zorder>
   <zorder>queue_1</zorder>
   <zorder>queue_0</zorder>
   <zorder>storeAddr_0</zorder>
   <zorder>storeAddr_1</zorder>
   <zorder>x_1</zorder>
   <zorder>x_2</zorder>
   <zorder>x_3</zorder>
   <zorder>x_0</zorder>
   <zorder>addsOp_0</zorder>
   <zorder>addsOp_1</zorder>
   <zorder>addsOp_2</zorder>
   <zorder>addsVj_0</zorder>
   <zorder>addsVj_1</zorder>
   <zorder>addsVj_2</zorder>
   <zorder>addsVk_0</zorder>
   <zorder>addsVk_1</zorder>
   <zorder>addsVk_2</zorder>
   <zorder>mulsOp_1</zorder>
   <zorder>mulsVj_0</zorder>
   <zorder>mulsVj_1</zorder>
   <zorder>mulsVk_0</zorder>
   <zorder>mulsVk_1</zorder>
   <zorder>storeAddr_2</zorder>
   <zorder>storeAddr_3</zorder>
   <zorder>storeAddr_4</zorder>
   <zorder>storeReg_0</zorder>
   <zorder>storeReg_1</zorder>
   <zorder>storeReg_2</zorder>
   <zorder>storeReg_3</zorder>
   <zorder>storeReg_4</zorder>
   <zorder>memoryTable</zorder>
   <zorder>portTable</zorder>
   <zorder>x_4</zorder>
   <zorder>x_5</zorder>
   <zorder>xq_0</zorder>
   <zorder>xq_1</zorder>
   <zorder>xq_2</zorder>
   <zorder>xq_3</zorder>
   <zorder>xq_4</zorder>
   <zorder>xq_5</zorder>
   <zorder>mulsOp_0</zorder>
   <zorder>add_0</zorder>
   <zorder>add_1</zorder>
   <zorder>add_2</zorder>
   <zorder>mul_0</zorder>
   <zorder>mul_1</zorder>
   <zorder>compileButton</zorder>
   <zorder>codeTextEdit</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1270</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
